go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Jeffail/tunny v0.1.4 // indirect
 	github.com/StackExchange/wmi v1.2.1 // indirect
 	github.com/antonmedv/expr v1.8.9
+    github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20210826220005-b48c857c3a0e // indirect
```

Comment:
need alignment

Reply:
Already aligned in fact. Github render's issue caused this.